catalogue/webapp/test/e2e/actions/common.ts

```diff
@@ -0,0 +1,41 @@
+export async function fillInputAction(
+  selector: string,
+  text: string
+): Promise<void> {
+  await page.fill(selector, text);
+}
+
+export async function pressEnterAction(selector: string): Promise<void> {
+  await page.press(selector, 'Enter');
```

Comment:
I personally still don't see these abstractions as necessary as the implementation is the same ot simpler than the abstraction.

Could you run me through it again why we want them?

Reply:
i can take this out. and replace this with the default